i-VRESSE/workflow-builder
# Use molecule filename as index

Fixes #57 
Fixes #76

TODO
* [x] lookup array as ui:indexable value
* [x] after molecule files have changed the uiSchemas with the ui:indexable parameter with maxItemsFrom > moleculeinputpaths in schema should be updated

Previous PR #104 was merged prematurly, this PR continues work.

To test:
1. In haddock3-download app
    1. upload pdb file into global parameters
    2. Add `mdref` node
    3. In form of `mdref` expand molecules subform
    4. Press `+` button on mol_shape parameter
    5. The filename of the uploaded pdb file should show as first column
1. In storybook of form package `yarn storybook` + http://localhost:6007/
    1.  The Array and TableField have stories with indexes.

Due to formatting with prettier and `yarn lint --fix` there are changes that are not relevant.
